src/pages/PostPage.tsx

```
@@ -1,51 +1,92 @@
 
+import { useCommentsQuery } from "@/features/comment/model/useCommentsQuery";
+import { useCreateCommentMutation } from "@/features/comment/model/useCreateCommentMutation";
+import { useDeleteCommentMutation } from "@/features/comment/model/useDeleteCommentMutation";
 import { useMyProfileQuery } from "@/features/member/model/useMyProfileQuery";
 import { normalizePostDomainError } from "@/features/post/model/postDomainError";
 import { resolvePostDomainErrorMessage } from "@/features/post/model/postDomainErrorMessage";
 import { mapPostDetailToPost } from "@/features/post/model/postUiMappers";
 import { usePostDetailQuery } from "@/features/post/model/usePostDetailQuery";
 import { useTogglePostLikeMutation } from "@/features/post/model/useTogglePostLikeMutation";
-import { MOCK_COMMENTS } from "@/mocks/detailPost";
 import { buildPostShareUrl } from "@/shared/lib/share-url";
```

ESLint

[error] There should be no empty line within import group

(import/order)